spec/controllers/posts_controller_spec.rb

```diff
@@ -5,10 +5,10 @@
 
 RSpec.describe PostsController, type: :controller do
 
-  let(:my_topic) { Topic.create!(name: RandomData.random_sentence, description: RandomData.random_paragraph)}
-  let(:my_user) { User.create!(name: "Bloccit User", email: "user@bloccit.com", password: "helloworld")}
-  let(:post_moderator) { User.create!(name: "Moderator user", email: "moderator@bloc.io", password: "helloworld", role: :moderator)}
-  let(:my_post) { my_topic.posts.create!(title: RandomData.random_sentence, body: RandomData.random_paragraph, user: my_user)}
+  let(:my_topic) { Topic.create!(name: RandomData.random_sentence, description: RandomData.random_paragraph) }
+  let(:my_user) { User.create!(name: "Bloccit User", email: "user@bloccit.com", password: "helloworld") }
+  let(:post_moderator) { User.create!(name: "Moderator user", email: "moderator@bloc.io", password: "helloworld", role: :moderator) }
+  let(:my_post) { my_topic.posts.create!(title: RandomData.random_sentence, body: RandomData.random_paragraph, user: my_user) }
 
```

Comment:
Here, I simply added a space before the final `}`, to match the existing space after the leading `{`. It improves readability and overall "cleanness" :+1: